spec/unit/outputs/kafka_spec.rb

```diff
@@ -141,15 +144,16 @@
 
             # Return a custom Future that will raise an exception to simulate a Kafka send() problem.
             future = java.util.concurrent.FutureTask.new { raise org.apache.kafka.common.errors.TimeoutException.new("Failed") }
-            future.run
-            future
           else
-            m.call(*args)
+            success = args
```

Comment:
If we aren't validating the contents beyond truthiness, setting them to `args` (which will be an array containing the args sent to our producer) adds unnecessary details. We may as well set it to `true` directly.